lib/meadow_web/schema/types/data/batch_types.ex

```diff
@@ -82,6 +82,7 @@ defmodule MeadowWeb.Schema.Data.BatchTypes do
     field :box_number, list_of(:string)
     field :caption, list_of(:string)
     field :catalog_key, list_of(:string)
+    field :date_created, list_of(:edtf_date_input)
```

Comment:
Happy to hear your thoughts especially if I'm missing something, but my take is that `:edtf_date_input` is probably overly complicated if all we're going to input is a string here. Would probably just go with `list_of(:string)` and keep the `list_of(:edtf_date_entry)` on the `:work_descriptive_metadata` object.

Reply:
I was thinking it might make it easier on the front end to send/display an object (just w/out one of the fields on the input case) ?

Reply:
It was more the input object itself that threw me a little bit so I had to dig into the docs, unless we need something more complicated in the frontend `createWork(descriptiveMetadata:{dateCreated:"?2016-?05-31"},...)` seems more natural to me than `createWork(descriptiveMetadata:{dateCreated:{edtfDate:"?2016-?05-31"}},...)`. I could get used to the latter, but just throwing out my $.02.

Reply:
I do hear you. And I agree with what you are saying - it's just I thought it might be easier for the front end (similar to what we do for coded terms) that they get back from the API  queries and mutations: `dateCreated: {edtf_date: "1975-07-01", humanized_date: "Tue, 01 Jul 1975"}` and then they can turn around and basically send the input object in the same format (minus the humanized) w/out having to reformat it.  -- Also not totally sure how it will all play out - but it's possible we might be able to just optionally take the humanized form from the front end/API and save a lookup on the backend if it's provided.....

I'd like to leave it for now and see how the front end evolves. If it turns out that it's overkill, switch it to a string.....